src/analyze/analyze-verify.h

```
@@ -6,5 +6,17 @@
 #include "execute.h"
 #include "path-lookup.h"
 
+typedef enum ReturnErrorOn {
```

I think this should be called FailVerificationOn or so

---

This was changed as such following feedback - @keszybz are you OK with `FailVerificationOn` instead of `ReturnErrorOn`?

---

Yeah, I think 'FailVerificationOn' is better.